client/lib/analytics/utils.js

```diff
@@ -86,6 +86,21 @@ export function hashPii( data ) {
 		.digest( 'hex' );
 }
 
+/**
+ * Returns the current user email after normalizing it (lowercase without spaces) or `false` if no email or user is available.
+ *
+ * @param {Object} user The current user
+ * @return {false|string} The current user email after normalization
+ */
+export function getNormalizedHashedUserEamil( user ) {
+	const currentUser = user.get();
+	if ( currentUser && currentUser.email ) {
+		return hashPii( currentUser.email.toLowerCase().replace( /\s/g, '' ) );
```

Comment:
I would also expect we would not have emails with spaces in them but when Alexandru originally implemented this code he included this check as well so I left it as is since I haven't actually checked that we do not need this and he might have seen some cases of spaces creeping in? Not sure.